core/controllers/base.py

```diff
@@ -504,7 +550,15 @@ def init_csrf_secret(cls):
 
     @classmethod
     def _create_token(cls, user_id, issued_on):
-        """Creates a digest (string representation) of a token."""
+        """Creates a new CSRF token.
+
+        Args:
+            user_id: str. The user_id for whom the token is generated.
+            issued_on: The time while generating token.
```

Comment:
Type information?

For description, consider: `The timestamp at which the token was issued.` "while" indicates a period of a time as opposed to a specific moment in time.